com.stansassets.scene-management/Editor/Visualizer/View/StateStackVisualizerViewDark.uss

```diff
@@ -0,0 +1,12 @@
+﻿.root {
+    background-color: #4d4d4d;
+}
+
+.stack-item {
+    border-color: rgb(0, 0, 0);
+    background-color: rgb(56, 56, 56);
+}
+
+.stack-item-active {
+    border-color: rgb(44, 191, 39);
+}
```

Comment:
Instead of doing same styles with different colors - we can make 2 versions of variables. Poke me and I will show you.